mne/data/html_templates.py

```diff
@@ -59,9 +59,13 @@
 
 raw_template = Template("""
 {{info_repr[:-9]}}
+    <tr>
+        <th>Filenames</th>
+        <td>{{', '.join(filenames)}}</td>
+    </tr>
     <tr>
         <th>Measurement time range</th>
```

Comment:
Do you think we could change this to simply "Measurement time"? Would be slightly more concise … @agramfort WDYT?